GoogleSignIn/Sources/GIDToken.m

```diff
@@ -25,7 +25,7 @@
 @implementation GIDToken
```

Comment:
Need to use `NS_ASSUME_NONNULL_BEGIN` / `NS_ASSUME_NONNULL_END` for this implementation.

Reply:
Done.